rero_ils/modules/mef_persons/jsonschemas/persons/mef_person-v0.0.1.json

```diff
@@ -21,6 +21,10 @@
       "title": "Library ID",
```

Comment:
This file is not used anymore as it is used for `records` store in the DB. However, we still need to check if it work without this file.

Reply:
TODO in task cleaning